litr-demo/src/main/java/com/linkedin/android/litr/demo/data/AudioTarget.java

```diff
@@ -22,6 +23,7 @@ public AudioTarget() {
     @Override
     public void onCheckedChanged(CompoundButton button, boolean checked) {
         shouldTranscodeAudio = checked;
+        shouldKeepTrack = checked;
```

Comment:
While shouldTranscodeAudio true then shouldKeepTrack true sounds right, shouldTranscodeAudio false then shouldKeepTrack false seems confusing.

Reply:
I think shouldTranscodeAudio=true && shouldKeepTrack=false sounds even more ridiculous (transcode and throw away)
We should come up with a better API for this. Probably something that will more closely mimic MediaTransformer API. For now, I don't care about this too much, but soon sample app has to be refactored to have more flexible data classes.

Reply:
Need a better API otherwise it's confusing.